pushapkscript/googleplay.py

```diff
@@ -61,3 +68,15 @@ def _get_play_config(context, channel):
     except KeyError:
         raise TaskVerificationError('Channel "{}" does not exist in the configuration of this instance.\
     Are you sure you allowed to push such APK?'.format(channel))
+
+
+def is_allowed_to_push_to_google_play(context):
+    channel = extract_channel(context.task)
+    return channel in _CHANNELS_AUTHORIZED_TO_REACH_GOOGLE_PLAY
+
+
+def will_commit_transaction(context):
```

Comment:
Tiny nit: to be honest, this function name is a bit biased to me. It assumes that you will actually assume the transaction, rather than a question mark there. Sadly I don't have other suggestion rather than `should_transaction_commit` but feel free to ignore this comment if you think this is sidetracking.

Reply:
Great point! I renamed this function to `should_commit_transaction()`